easybuild/easyconfigs/b/bx-python/bx-python-0.8.8-foss-2019b-Python-3.7.4.eb

```diff
@@ -0,0 +1,36 @@
+easyblock = 'PythonBundle'
+
+name = 'bx-python'
+version = '0.8.8'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://github.com/bxlab/bx-python'
+description = """The bx-python project is a Python library and associated set of scripts to allow for rapid
+ implementation of genome scale analyses."""
+
+toolchain = {'name': 'foss', 'version': '2019b'}
+
+dependencies = [
+    ('Python', '3.7.4'),
+    ('LZO', '2.10'),
+    ('SciPy-bundle', '2019.10', versionsuffix),
+]
+
+prebuildopts = "export CPATH=$EBROOTLZO/include/lzo:$CPATH && "
```

Comment:
@fizwit this is a pre-existing issue: this line does not affect extensions, so it does not do anything. see below for a proper fix.

Reply:
prebuildopts relocated to exts_list